kensu/pyspark/spark_connector.py

```diff
@@ -0,0 +1,839 @@
+import os
```

Comment:
for reference here's a diff from non-OSS code: https://gist.github.com/vidma/f8a5ca99cb3962506e514825de7c9f72/revisions

generally diff looks good and contains only very minor changes (not checking cleanup opportunities now) 👍